docs/tutorials/basic-usage.md

```diff
@@ -23,16 +23,16 @@ Basic usage of Shaka Player is very easy:
   <body>
     <video id="video"
            width="640"
-           poster="//shaka-player-demo.appspot.com/assets/poster.jpg"
-           controls autoplay></video>
+           poster="/assets/poster.jpg"
+           controls autoplay playsinline></video>
```

Comment:
A link to playsinline docs in the commit message would be helpful.  You mention that it's required for iOS, which we do not currently support.  If there are other reasons for this, we need context.

Reply:
Since iOS is not supported yet*, I'll revert for now.